src/core_atmosphere/physics/mpas_atmphys_driver_pbl.F

```diff
@@ -282,56 +329,62 @@ subroutine pbl_from_MPAS(configs,mesh,sfc_input,diag_physics,tend_physics,its,it
 !local pointers for MYNN scheme:
  real(kind=RKIND),pointer:: len_disp
  real(kind=RKIND),dimension(:),pointer  :: meshDensity
- real(kind=RKIND),dimension(:),pointer  :: ch,qsfc,qcg,rmol,skintemp
- real(kind=RKIND),dimension(:,:),pointer:: cov,qke,qsq,tsq,sh3d,tke_pbl,qke_adv,el_pbl
+ real(kind=RKIND),dimension(:),pointer  :: ch,qsfc,rmol,skintemp
+ real(kind=RKIND),dimension(:,:),pointer:: cov,qke,qsq,tsq,sh3d,sm3d,tke_pbl,qke_adv,el_pbl
+ real(kind=RKIND),dimension(:,:),pointer:: cldfrac_bl,qc_bl,qi_bl
+ real(kind=RKIND),dimension(:,:),pointer:: edmf_a,edmf_ent,edmf_qc,edmf_qt,edmf_thl,edmf_w
+ real(kind=RKIND),dimension(:,:),pointer:: sub_thl,sub_qv,det_thl,det_qv
 
 !-----------------------------------------------------------------------------------------------------------------
 
  call mpas_pool_get_config(configs,'config_pbl_scheme',pbl_scheme)
 
- call mpas_pool_get_array(diag_physics,'hfx'   ,hfx   )
- call mpas_pool_get_array(diag_physics,'hpbl'  ,hpbl  )
- call mpas_pool_get_array(diag_physics,'qfx'   ,qfx   )
- call mpas_pool_get_array(diag_physics,'ust'   ,ust   )
- call mpas_pool_get_array(diag_physics,'wspd'  ,wspd  )
- call mpas_pool_get_array(diag_physics,'znt'   ,znt   )
- call mpas_pool_get_array(diag_physics,'delta' ,delta )
- call mpas_pool_get_array(diag_physics,'wstar' ,wstar )
+ call mpas_pool_get_array(diag_physics,'hfx' ,hfx )
+ call mpas_pool_get_array(diag_physics,'hpbl',hpbl)
+ call mpas_pool_get_array(diag_physics,'qfx' ,qfx )
+ call mpas_pool_get_array(diag_physics,'ust' ,ust )
+ call mpas_pool_get_array(diag_physics,'wspd',wspd)
+ call mpas_pool_get_array(diag_physics,'znt' ,znt )
 
- call mpas_pool_get_array(sfc_input   ,'xland' ,xland )
+ call mpas_pool_get_array(tend_physics,'rthratenlw',rthratenlw)
+ call mpas_pool_get_array(tend_physics,'rthratensw',rthratensw)
+
+ call mpas_pool_get_array(sfc_input,'xland',xland)
 
  do j = jts,jte
  do i = its,ite
     !from surface-layer model:
-    hfx_p(i,j)    = hfx(i)
-    hpbl_p(i,j)   = hpbl(i)
-    qfx_p(i,j)    = qfx(i)
-    ust_p(i,j)    = ust(i)
-    wspd_p(i,j)   = wspd(i)
-    xland_p(i,j)  = xland(i)
-    kpbl_p(i,j)   = 1
-    znt_p(i,j)    = znt(i)
-    delta_p(i,j)  = delta(i)
-    wstar_p(i,j)  = wstar(i)
+    hfx_p(i,j)   = hfx(i)
+    hpbl_p(i,j)  = hpbl(i)
+    qfx_p(i,j)   = qfx(i)
+    ust_p(i,j)   = ust(i)
+    wspd_p(i,j)  = wspd(i)
+    xland_p(i,j) = xland(i)
+    kpbl_p(i,j)  = 1
```

Comment:
@joeolson42 checking that line 363 is okay.

Reply:
yeah, that is OK.